tests/Http/Api/ReportbackApiTest.php

```diff
@@ -2,30 +2,34 @@
 
 namespace Tests\Http\Api;
 
+use Tests\TestCase;
 use Rogue\Models\Post;
 use Rogue\Models\Signup;
-use Tests\BrowserKitTestCase;
 
-class ReportbackApiTest extends BrowserKitTestCase
+class ReportbackApiTest extends TestCase
 {
+    /**
+     * Test the "legacy" style reportback endpoint.
+     *
+     * GET api/v1/reportbacks
+     */
     public function testGetReportbacks()
     {
-        $signups = factory(Signup::class, 10)->create();
+        factory(Signup::class, 10)->create()
+            ->each(function (Signup $signup) {
+                // Give each signup a post and give it status of "accepted"
+                $post = factory(Post::class)->make();
+                $post->campaign_id = $signup->campaign_id;
+                $post->northstar_id = $signup->northstar_id;
+                $post->status = 'accepted';
```

Comment:
Probably out of scope but I think we can just create an accepted post here

Reply:
Oh yeah! Done in 073fc5c3c4f9261d677de62e77326a7cd2160cc9.